os.go

```diff
@@ -34,7 +33,7 @@ func (root osFS) resolve(path string) string {
 	// If it did, the result of filepath.Join would be outside the
 	// tree rooted at root.  We probably won't ever see a path
 	// with .. in it, but be safe anyway.
-	path = pathpkg.Clean("/" + path)
```

Comment:
I've also read a bit of the code in `os.go` and I'm not convinced it actually works well on windows, which may be your issue. It is surprising that no one has really used the VFS stuff on windows, but is possible.